src/anim/binder/default-anim-binder.js

```diff
@@ -78,25 +78,18 @@ class DefaultAnimBinder {
                 return DefaultAnimBinder.createAnimTarget(func, 'vector', 3, node, 'localScale');
             },
 
-            'weights': function (node) {
+            'weight': function (node, weightName) {
                 const meshInstances = findMeshInstances(node);
                 if (meshInstances) {
-                    const morphInstances = [];
                     for (let i = 0; i < meshInstances.length; ++i) {
                         if (meshInstances[i].node.name === node.name && meshInstances[i].morphInstance) {
-                            morphInstances.push(meshInstances[i].morphInstance);
+                            const morphInstance = meshInstances[i].morphInstance;
+                            const func = function (value) {
```

Comment:
Any reason not to use an arrow function here?

Reply:
Nope, done.